hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1678,6 +1681,13 @@ message SnapshotDiffRequest {
   required string toSnapshot = 4;
 }
 
+message DeleteSnapshotRequest {
+  optional string volumeName = 1;
+  optional string bucketName = 2;
+  optional string snapshotName = 3;
+  optional uint64 deletionTime = 4;
```

Comment:
`deletionTime` is here to be synced across OMs. Otherwise the deletionTime in the SnapshotInfo entry for a snapshot won't be guaranteed synced if each OM generates the timestamp themselves.